lib/routes.js

```diff
@@ -219,11 +219,17 @@ module.exports = function(self, options) {
         // Confusion to the enemy
         return res.status(404).send('not found');
       }
-      req.session.workflowMode = (req.body.mode === 'draft') ? 'draft' : 'live';
-      if (req.body.mode === 'draft') {
+      req.session.workflowMode = (req.body.mode === 'draft' || req.body.mode === 'preview') ? 'draft' : 'live';
+      if (req.body.mode === 'preview') {
+        req.body.mode = 'draft';
+        req.session.isPreview = true;
```

Comment:
If the preview mode is chosen by the users a flag(**isPreview**) is set to **session** object, which is used afterwards to disable the apos-ui editor controls in the front-end. Every other variable in the object will be same as that of  the **draft** mode.

Reply:
`req.session.workflowPreview`, as well.